.changeset/empty-singers-heal.md

```diff
@@ -0,0 +1,13 @@
+---
+'wingman-fe': minor
```

Comment:
Isn't this technically a breaking change?
```suggestion
'wingman-fe': major
```

Reply:
We've been cheating semver by staying on 0.x, but it's probably time for us to take things seriously.